content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+---
+weight: 50
+title: Command Reference
+description: "Command reference for the Crossplane CLI"
+---
+
+<!-- vale Google.Headings = NO -->
+The `crossplane` CLI provides utilities to make using Crossplane easier. 
+
+Read the [Crossplane CLI overview]({{<ref "../cli">}}) page for information on 
+installing `crossplane`.
+
+## Global flags
+The following flags are available for all commands.
+
+{{< table "table table-sm table-striped">}}
+| Short flag | Long flag   | Description                  |
+|------------|-------------|------------------------------|
+| `-h`       | `--help`    | Show context sensitive help. |
+| `-v`       | `--version` | Print version and exit.      |
+|            | `--verbose` | Print verbose output.        |
+{{< /table >}}
+
+## xpkg
+
+The `crossplane xpkg` commands create, install and update Crossplane
+[packages]({{<ref "../concepts/packages">}}) as well as enable authentication
+and publishing of Crossplane packages to a Crossplane package registry. 
+
+### xpkg build
+
+Using `crossplane xpkg build` provides automation and simplification to build 
+Crossplane packages.  
+
+The Crossplane CLI creates a 
+single YAML file from a directory of YAML files and combines them as an 
+[OCI container image](https://opencontainers.org/).  
+
+The CLI applies the required annotations and values to meet the 
+[Crossplane XPKG specification](https://github.com/crossplane/crossplane/blob/master/contributing/specifications/xpkg.md).
+
+The `crossplane` CLI supports building 
+[configuration]({{< ref "../concepts/packages" >}}),
+[function]({{<ref "../concepts/composition-functions">}}) and 
+[provider]({{<ref "../concepts/providers" >}}) package types. 
+
+
+#### Flags
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                            | Description                    |
+| ------------ | -------------                        | ------------------------------ |
+|              | `--embed-runtime-image-name=NAME`    |  The image name and tag of a container to include in the package. Only for provider and function packages. |
+|              | `--embed-runtime-image-tarball=PATH` |  The filename of a container to include in the package. Only for provider and function packages.                              |
```

Comment:
Should we explain to the user why we're including these containers (nit: technically it's an image - containers are running images) in the package?

We include them so that the resulting package can be used both by the package manager to install the package, and by Kubernetes to run the package's "runtime". The runtime could be a provider pod, or a function pod.

Reply:
No, I think I wrote this comment before I noticed that section and forgot to remove it. Thanks!